src/app/admin-panel/dashboard/dashboard.component.scss

```diff
@@ -229,22 +235,308 @@ $background-gradient:
   border-bottom: 0.05rem solid rgba(236, 236, 236, 0.64);
 }
 
-@media (min-width: 768px) and (max-width: 1024px) {
+@media (min-width: 1920px) {
+  .graph {
+    justify-content: space-around;
+    font-size: 1.27em;
+
+    .chart {
+      height: auto;
+    }
+
+    .chartOptions {
+      height: 24.9em;
+      mat-card-content {
+        form {
+          font-size: 1.5em;
+        }
+      }
+    }
+  }
+
+  .stats {
+    flex-grow: 3;
+    font-size: 1.4em;
+
+    mat-card {
+      margin: 0.7em;
+    }
+
+    .cardHeader {
+      font-size: 0.8em;
+
+      mat-icon {
+        margin: -0.7em 0 0 -0.75em;
+      }
+    }
+
+    mat-card-content {
+      font-size: 0.7em;
+    }
+
+    mat-card-actions {
+
+      button {
+        font-size: 0.7em;
+      }
+      mat-icon {
+        margin-left: 5.2em;
+      }
+    }
+  }
+}
+
+@media (min-width: 1601px) and (max-width: 1920px) {
+  .graph {
+    justify-content: space-around;
+    font-size: 1.27em;
+
+    .chart {
+      height: auto;
+    }
+
+    .chartOptions {
+      height: 24.9em;
+      mat-card-content {
+        form {
+          font-size: 1.5em;
+        }
+      }
+    }
+  }
+
+  .stats {
+    flex-grow: 3;
+    font-size: 1.4em;
+
+    mat-card {
+      margin: 0.7em;
+    }
+
+    .cardHeader {
+      font-size: 0.8em;
+
+      mat-icon {
+        margin: -0.7em 0 0 -0.75em;
+      }
+    }
+
+    mat-card-content {
+      font-size: 0.7em;
+    }
+
+    mat-card-actions {
+
+      button {
+        font-size: 0.7em;
+      }
+      mat-icon {
+        margin-left: 5.2em;
+      }
+    }
+  }
+}
+
+@media (min-width: 1367px) and (max-width: 1600px) {
+  .graph {
+    justify-content: space-around;
+    font-size: 1.06em;
+
+    .chart {
+      height: auto;
+    }
+
+    .chartOptions {
+
+      mat-card-header {
+
+      }
+
+      mat-card-content {
+
+        form {
+          font-size: 1.4em;
+          mat-form-field {
+
+            mat-label {
+
+            }
+
+            mat-select {
+            }
+            mat-error {
+            }
+          }
+
+          .graphType {
+
+            mat-radio-group {
+            }
+
+            label {
+
+            }
+
+          }
+
+          mat-radio-button {
+
+          }
+
+          button {
+
+          }
+        }
+      }
+    }
+  }
+
+  .stats {
+    flex-grow: 1;
+    font-size: 1.2em;
+
+    mat-card {
+      margin: 1em;
+    }
+
+    .cardHeader {
+      font-size: 0.8em;
+
+      mat-icon {
+
```

Comment:
why do we need empty selectors?

Reply:
Sorry, my bad, i have copied the structure of all document into media queries to have i possibility to edit it easier. Fixed!